src/plugins/network.ts

```diff
@@ -117,15 +118,25 @@ export class NetworkManager {
       step: this._currentStep,
       timestamp: getTimestamp(),
       url,
-      request,
       type,
       method,
-      requestSentTime: timestamp,
+      status: -1,
```

Comment:
We previously set this to `0`, also, why not `null`? It'd be great to have a comment somewhere nearby explaining these numeric values.

Reply:
Hmmm good point, status: 0 denotes a successful response when the browser was having CORS issue so to avoid mistaking it for the aborted requests, I have set it to `-1` which feels right. 

`null` would denote a separate type. Keeping `number` type seems better for status codes.